src/evict/evict_lru.c

```diff
@@ -1617,6 +1620,22 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
         goto retry;
     }
 
+#ifdef HAVE_DIAGNOSTIC
+    if (!F_ISSET(conn, WT_CONN_IN_MEMORY) && cache->evict_aggressive_score == WT_EVICT_SCORE_MAX &&
```

Comment:
I hope we never exceed `WT_EVICT_SCORE_MAX` (we shouldn't) but I tend to prefer the following:
```suggestion
    if (!F_ISSET(conn, WT_CONN_IN_MEMORY) && cache->evict_aggressive_score >= WT_EVICT_SCORE_MAX &&
```

Reply:
change as suggested.